.pre-commit-config.yaml

```diff
@@ -51,13 +51,10 @@ repos:
       - id: check-github-actions
 
   - repo: https://github.com/pappasam/toml-sort
-    rev: v0.20.1
+    rev: v0.22.1
     hooks:
       - id: toml-sort
-        args:
-          - --all
-          - --in-place
-          - --ignore-case
+        args: []
```

Comment:
I did check to confirm that an empty `args` array cannot be omitted from the `pre-commit` config without causing an error (it cannot be).